### PR TITLE
The ID column made non-sortable

### PR DESCRIPTION
**Pull request by kthorat-prog**

### CODE REVIEW

File: src/main/cliapp/src/containers/controlledVocabularyPage/ControlledVocabularyComponent.js [hunk 1]
```diff
@@ -392,7 +393,7 @@ export const ControlledVocabularyComponent = () => {
           key={col.field}
           field={col.field}
           header={col.header}
-          sortable={isEnabled}
+          sortable={(col.field == "id")? false : isEnabled}
```
Comment (oblodgett) [inline src/main/cliapp/src/containers/controlledVocabularyPage/ControlledVocabularyComponent.js]: Ahh this is a bug in this file... the code should be:
`sortable={{col.sortable}}`

**Review by oblodgett — CHANGES_REQUESTED** [review]

File: src/main/cliapp/src/containers/controlledVocabularyPage/ControlledVocabularyComponent.js (hunk 1 above)
Comment (kthorat-prog) [reply src/main/cliapp/src/containers/controlledVocabularyPage/ControlledVocabularyComponent.js]: Yes. Will check and update it. Thanks.

Comment (adamgibs) [inline src/main/cliapp/src/containers/controlledVocabularyPage/ControlledVocabularyComponent.js]: Maybe sortable on the <Column> component should just be set to col.sortable and set the sortable field on the ID column object to false?

Comment (oblodgett) [reply src/main/cliapp/src/containers/controlledVocabularyPage/ControlledVocabularyComponent.js]: Not sure I understand this comment.

Comment (kthorat-prog) [reply src/main/cliapp/src/containers/controlledVocabularyPage/ControlledVocabularyComponent.js]: Please review the File changed now.

Comment (adamgibs) [reply src/main/cliapp/src/containers/controlledVocabularyPage/ControlledVocabularyComponent.js]: @oblodgett I think we just said the same thing. I just didn't notice your comment.

**Review by oblodgett — APPROVED** [review]

**Pull request merged**